test/functional/rpc_deprecated.py

```diff
@@ -7,21 +7,19 @@
 
 class DeprecatedRpcTest(BitcoinTestFramework):
     def set_test_params(self):
-        self.num_nodes = 2
+        self.num_nodes = 1
         self.setup_clean_chain = True
-        self.extra_args = [[], ['-deprecatedrpc=bumpfee']]
+        self.extra_args = [[]]
 
     def run_test(self):
         # This test should be used to verify correct behaviour of deprecated
-        # RPC methods with and without the -deprecatedrpc flags. For example:
+        # RPC methods without the -deprecatedrpc flag. For example:
         #
-        # In set_test_params:
-        # self.extra_args = [[], ["-deprecatedrpc=generate"]]
-        #
-        # In run_test:
         # self.log.info("Test generate RPC")
         # assert_raises_rpc_error(-32, 'The wallet generate rpc method is deprecated', self.nodes[0].rpc.generate, 1)
-        # self.generate(self.nodes[1], 1)
+        #
+        # Please ensure that the -depgrecatedrpc flag for this RPC method is
+        # used in anther test where the RPC functionality is actually tested.
 
         self.log.info("No tested deprecated RPC methods")
```

Comment:
Nit:
```
 self.log.info("No tested currently deprecated RPC methods")
```

Reply:
Changed this but with slightly different wording